README.md

```diff
@@ -32,6 +33,31 @@ To build the app from source:
 - Eclipse 3.6.2 or higher (Eclipse for Android recommended)
 - Android Support Library
 
+## Release Notes
+
+### 3.0.0
+
+#### What's New in Squad Leader 3.0.0
+* Uses ArcGIS Runtime 10.2.4.
+  * If you built Squad Leader with a previous version of ArcGIS Runtime, you need to follow the step in [Building with Eclipse](#building-with-eclipse) below about copying the MIL-STD-2525C symbol dictionary into your clone. ArcGIS Runtime 10.2.4 will not work with older versions of the symbol dictionary.
+* Requires Android 4.0.1 or higher
+* Viewshed analysis
+    * Requires Android 4.1 (Android API level 16) or higher
+    * Viewshed requires elevation data on the device. See the [User Guide](documentation/UserGuide.md) for configuration details.
+* GeoMessage management (e.g. chem lights, spot reports, and position reports):
+    * The user can remove all messages of a certain type or all messages of all types from the map, with the option to send a remove message for messages created by this user to other clients.
+    * The user can modify or remove a chem light.
+    * The app recognizes the new "removeall" GeoMessage action, which removes all messages of a certain type:
+    ```
+        <geomessage v="1.0">
+            <_type>chemlight</_type>
+            <_action>removeall</_action>
+            <_id>{2e50e7ea-c76a-11e4-8731-1681e6b88ec1}</_id>
+        </geomessage>
+    ```
+* Using a client ID and license string that are configurable in strings.xml. Note that the current code only requires a Runtime Basic license, so the license string is optional.
```

Comment:
@csmoore, that is correct. At Runtime 10.2.4 and 10.2.5 for Android, offline viewshed is a beta capability that is included in the Runtime Basic license.